build.sbt

```diff
@@ -175,32 +176,32 @@ lazy val commonWDS = Seq(
   webpackBundlingMode in fastOptJS := BundlingMode.LibraryOnly(),
   webpackBundlingMode in fullOptJS := BundlingMode.Application,
   test := {},
-  emitSourceMaps := false,
+  scalaJSLinkerConfig in (Compile, fastOptJS) ~= { _.withSourceMap(false) },
+  scalaJSLinkerConfig in (Compile, fullOptJS) ~= { _.withSourceMap(false) },
   // NPM libs for development, mostly to let webpack do its magic
   npmDevDependencies in Compile ++= Seq(
     "postcss-loader"                     -> "3.0.0",
-    "autoprefixer"                       -> "9.7.1",
-    "url-loader"                         -> "2.2.0",
-    "file-loader"                        -> "4.2.0",
-    "css-loader"                         -> "3.2.0",
-    "style-loader"                       -> "1.0.0",
-    "less"                               -> "2.7.2",
-    "less-loader"                        -> "4.1.0",
+    "autoprefixer"                       -> "9.7.6",
+    "url-loader"                         -> "4.1.0",
+    "file-loader"                        -> "6.0.0",
+    "css-loader"                         -> "3.5.3",
+    "style-loader"                       -> "1.2.1",
+    "less"                               -> "3.11.1",
+    "less-loader"                        -> "6.1.0",
     "webpack-merge"                      -> "4.2.2",
-    "mini-css-extract-plugin"            -> "0.8.0",
-    "webpack-dev-server-status-bar"      -> "1.1.0",
+    "mini-css-extract-plugin"            -> "0.9.0",
+    "webpack-dev-server-status-bar"      -> "1.1.2",
     "cssnano"                            -> "4.1.10",
-    "uglifyjs-webpack-plugin"            -> "2.2.0",
-    "html-webpack-plugin"                -> "3.2.0",
+    "terser-webpack-plugin"              -> "3.0.1",
```

Comment:
Did you try with and without uglyfiyng just to check it really makes a difference. A few times `uglifyjs` hasn't worked as expected

Reply:
Never occurred to me not to uglify (should we be saying terserify now?). Let me try and compare.

Reply:
It is just some older version of uglyfyjs wouldn't work with scala.js so I took it away, A later version did work and I restored it

Reply:
ok, uglify seems to be deprecated now in favor of terser, which seems to do a much better job

Reply:
I wish best in kind JS tools would last for a few weeks, uglify is probably the third minifier I've replaced of a saga of obsoleted tool